src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/Snapshot/Dependency.cs

```diff
@@ -296,13 +233,27 @@ public override string ToString()
             return Id;
         }
 
+        private static string GetAlias(IDependency dependency)
+        {
+            var path = dependency.OriginalItemSpec ?? dependency.Path;
+            if (string.IsNullOrEmpty(path) || path.Equals(dependency.Caption, StringComparison.OrdinalIgnoreCase))
+            {
+                return dependency.Caption;
+            }
+            else
+            {
+                return string.Format(CultureInfo.CurrentCulture, "{0} ({1})", dependency.Caption, path);
+            }
+        }
+
         private static string Normalize(string id)
         {
             return id.Replace('.', '_').Replace('/', '\\');
         }
 
         public static string GetID(ITargetFramework targetFramework, string providerType, string modelId)
```

Comment:
right, will add a check

---
In reply to: [111637352](https://github.com/dotnet/project-system/pull/1991#discussion_r111637352) [](ancestors = 111637352)